lib/locales/ta.ts

```diff
@@ -0,0 +1,998 @@
+    oleksiy_zarytskyi_priest: 'முத்திப்பேறுபெற்ற ஒலெக்சி சாரிட்ஸ்கி - அருடபணியாளர், மறைசாட்சி',
+    olga_of_kiev: 'புனித ஒல்கா',
+    oliver_plunket_bishop: 'புனித ஒலிவர் ப்ளங்கட் - ஆயர், மறைசாட்சி',
+    otteran_of_iona_monk: 'புனித ஒடெரன் - துறவி',
+    otto_of_bamberg_bishop: 'புனித ஒட்டோ பம்பெர்க் - ஆயர்',
+    audoen_of_rouen_bishop: 'புனித ஒவென் - ஆயர்',
+    our_lady_help_of_christians: 'கிறிஸ்தவர்களின் அடைக்கல மாதா',
+    our_lady_mediatrix_of_all_grace: 'அனைத்து கிருபைகளின் மத்தியஸ்தகராகிய மாதா',
+    our_lady_mother_of_christian_unity: 'கிறிஸ்தவர்களின் ஒற்றுமையின் மாதா',
+    our_lady_mother_of_divine_providence_patroness_of_puerto_rico: 'இறைபாதுப்பின் மாதா - பியொட்ரோ ரிக்கோவின் பாதுகாவலி',
+    our_lady_of_aparecida_patroness_of_brazil: 'அபரெசிடாவின் மாதா - பிரெசில் நாட்டின் பாதுகாவலி',
+    our_lady_of_bethlehem: 'பெத்லெகெமின் மாதா',
+    our_lady_of_china: 'சீனாவின் மாதா',
+    our_lady_of_czestochowa: 'செஸ்டொசொவாவின் மாதா',
+    our_lady_of_fatima: 'பாத்திமா மாதா',
+    our_lady_of_good_counsel: 'நல்லாலோசனை மாதா',
+    our_lady_of_guadalupe: 'குவாடலுபே மாதா',
+    our_lady_of_guadalupe_patroness_of_the_americas: 'குவாடலுபே மாதா - அமெரிக்க கண்டங்களின் பாதுகாவலி',
+    our_lady_of_guadalupe_patroness_of_the_philippines: 'குவாடலுபே மாதா - பிலிப்பைன்ஸ் தீவுகளின் பாதுகாவலி',
+    our_lady_of_hungary_patroness_of_hungary: 'ஹங்கேரியின் மாதா - ஹங்கேரியின் பாதுகாவலி',
+    our_lady_of_itati: 'இடாடியின் மாதா',
+    our_lady_of_knock: 'நொக்கின் மாதா',
+    our_lady_of_lanka: 'இலங்கை மாதா',
+    our_lady_of_lebanon: 'லெபனானின் மாதா',
+    our_lady_of_loreto: 'லொரெடா மாதா',
+    our_lady_of_lourdes: 'லூர்து மாதா',
+    our_lady_of_lujan_patroness_of_argentina: 'லுஜன் மாதா - அர்ஜெண்டினாவின் பாதுபாகவலி',
+    our_lady_of_madhu: 'மடு மாதா',
+    our_lady_of_marija_bistrica: 'மரிய பிஸ்ரிசா மாதா',
+    our_lady_of_mercy: 'இரக்கத்தின் மாத',
+    our_lady_of_mount_carmel: 'காமேலேற் மாதா',
+    our_lady_of_mount_carmel_mother_and_queen_of_chile: 'காமாலெற் மாதா - சிலியின் அன்னை, அரசி',
+    our_lady_of_perpetual_help: 'சதா சகாய மாதா',
+    our_lady_of_sorrows: 'வியாகுல மாதா',
+    our_lady_of_sorrows_patroness_of_slovakia: 'வியாகுல மாதா - சிலோவாக்கியாவின் பாதுகாவலி',
+    our_lady_of_the_discovery_of_the_hidden_christians: 'மறைந்திருக்கும் கிறிஸ்தவர்களை கண்டுபிடிக்கும் மாதா',
+    our_lady_of_the_gate_of_dawn: 'விடியலின் வாயில் மாதா',
+    our_lady_of_the_miraculous_medal: 'புதுமைப்பதக்க மாதா',
+    our_lady_of_the_pillar: 'தூண் மாதா',
+    our_lady_of_the_rosary: 'செபமாலை மாதா',
+    our_lady_of_the_valley: 'பள்ளத்தாக்குகளின் மாதா',
+    our_lady_of_vladimir: 'விளாடிமர் மாதா',
+    our_lady_of_walsingham: 'வெல்சிங்கம் மாதா',
+    our_lady_queen_of_peace: 'சமாதானத்தின் மாதா',
+    our_lady_queen_of_poland: 'போலந்தின் மாதா - போலந்தின் பாதுகாவலி',
+    our_lord_jesus_christ_king_of_the_universe: 'நமது ஆண்டவராகிய இயேசு கிறிஸ்து, நித்திய தலைமைக் குரு',
+    // our_lord_jesus_christ_king_of_the_universe: 'இயேசு கிறிஸ்து அனைத்துலக அரசர்',
```

Comment:
This translation was duplicated. 

@pvanthony, could you please indicate which one is more appropriate here? (FYI, it corresponds to the last Sunday of ordinary time)

Reply:
According to Google Translate, `நமது ஆண்டவராகிய இயேசு கிறிஸ்து, நித்திய தலைமைக் குரு` means `Our Lord Jesus Christ is the Eternal High Priest`, therefore, I’ll change the key name to `our_lord_jesus_christ_the_eternal_high_priest` and uncomment the other key.